app/test_split_integer.py

```diff
@@ -2,20 +2,35 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(36, 7)) == 36, (
+        f"Sum of parts should be equal to 36 "
+        f"but is {sum(split_integer(36, 7))}"
+    )
```

Comment:
```suggestion
    result = sum(split_integer(36, 7))
    assert result == 36, (
        f"Sum of parts should be equal to 36 "
        f"but is {result}"
    )
```

it's better to create a variable so you don't call the same function twice